src/utils/fiatToCrypto.ts

```diff
@@ -109,11 +136,23 @@ export const buildMtPelerinOptions = (code: string, address: string) => {
   return onRampOptions;
 };
 
+const getNetTypeFromChain = (chain: Chain) => {
+  if (chain === CHAIN.ETHEREUM) return 'mainnet';
+  if (chain === CHAIN.BINANCE) return 'bsc_mainnet';
+  if (chain === CHAIN.XDAI) return 'xdai_mainnet';
+  if (chain === CHAIN.OPTIMISM) return 'optimism_mainnet';
+  if (chain === CHAIN.ARBITRUM) return 'arbitrum_mainnet';
```

Comment:
Do we need to consider our testnet networks here?

Reply:
Is Mtpelerin testnet supported?

Reply:
I would assume no, but have you checked their docs?

Reply:
I checked doc. https://www.mtpelerin.com/currencies

Reply:
And are there testnets available? If not we can disregard this

Reply:
No testnets available.

Reply:
Resolved